keel-sql/src/main/resources/db/changelog/20190227-initial-schema.yml

```diff
@@ -53,8 +53,8 @@ databaseChangeLog:
             tableName: resource_state
             columns:
               - column:
-                  name: name
-                  type: varchar(255)
+                  name: uid
+                  type: char(26)
```

Comment:
Asher was of the opinion this was a better schema and I think it's more logical to use the primary key of the `resource` table.